src/silx/gui/data/DataViews.py

```diff
@@ -1792,7 +1844,25 @@ def createWidget(self, parent):
         widget.getPlot().getColormapAction().setColormapDialog(
             self.defaultColorDialog()
         )
+        self.__aggregationModeAction = AggregationModeAction(parent=widget)
+        widget.getPlot().toolBar().addAction(self.__aggregationModeAction)
+        self.__aggregationModeAction.sigAggregationModeChanged.connect(self._aggregationModeChanged)
         return widget
+    
+    def getAggregationModeAction(self) -> AggregationModeAction:
+        """Action toggling the aggregation mode action
+        """
+        return self.__aggregationModeAction
+
+    def _aggregationModeChanged(self):
+        plot = self.getWidget().getPlot()
+        item = plot._getItem("image")
+
+        if item is None:
+            return
+        
+        if isinstance(item, ImageDataAggregated):
+            item.setAggregationMode(self.getAggregationModeAction().getAggregationMode())
```

Comment:
Same here, the widget controlling the plot item is `ArrayImagePlot`, so the let it handle the aggregation mode as well (this would allow to enable/disable this action according to the displayed item image/scatter)